src/utils/pathHandling.ts

```diff
@@ -1,5 +1,9 @@
 import path from 'path';
 import type { FileSharePath } from '@/shared.types';
+import type { ProxiedPath } from '@/contexts/ProxiedPathContext';
+
+const PATH_DELIMITER = '/';
+const PROXY_BASE_URL = import.meta.env.VITE_PROXY_BASE_URL;
```

Comment:
In addition to adding documentation about adding this to an env file, we could throw a warning here if the import.meta.env... is undefined.